salmon/model.py

```diff
@@ -397,20 +381,22 @@ def score(self, X = None, y = None, adjusted = False, **kwargs):
     def _prediction_interval_width(self, X_new, alpha = 0.05):
         ''' Helper function for calculating prediction interval widths. '''
         mse = self.get_sse() / self.rdf
-        s_yhat_squared = (X_new.dot(self.cov_) * X_new).sum(axis = 1)
+        s_yhat_squared = (np.multiply(X_new.dot(self.cov_), X_new)).sum(axis = 1)
         s_pred_squared = mse + s_yhat_squared
 
         t_crit = stats.t.ppf(1 - (alpha / 2), self.rdf)
-
         return t_crit * (s_pred_squared ** 0.5)
 
     def _confidence_interval_width(self, X_new, alpha = 0.05):
         ''' Helper function for calculating confidence interval widths. '''
         _, p = X_new.shape
-        s_yhat_squared = (X_new.dot(self.cov_) * X_new).sum(axis = 1)
-        #t_crit = stats.t.ppf(1 - (alpha / 2), n-p)
-        W_crit_squared = p * stats.f.ppf(1 - (alpha / 2), p, self.rdf)
-        return (W_crit_squared ** 0.5) * (s_yhat_squared ** 0.5)
+        s_yhat_squared = (np.multiply(X_new.dot(self.cov_), X_new)).sum(axis = 1)
+        t_crit = stats.t.ppf(1 - (alpha / 2), self.rdf)
+        #W_crit_squared = p * stats.f.ppf(1 - (alpha / 2), p, self.rdf)
+        # print(W_crit_squared)
+        # print(p)
+        #return (W_crit_squared ** 0.5) * (s_yhat_squared ** 0.5)
```

Comment:
I imagine these were taken out because they didn't match the output from R? It's a special kind of confidence interval that accounts for multiple comparisons. We should probably keep it the way that you wrote in your commit though. 

Just be sure to take out the commented code.